src/assets/scss/FormField.scss

```diff
@@ -0,0 +1,125 @@
+.formField {
+  display: flex;
+  align-items: center;
+  padding-left: 1rem;
+  padding-right: 6rem;
+  padding-top: 1rem;
+  border-radius: 0.5rem;
+}
+
+.formField__label {
+  width:25%;
+  padding-right: 3rem;
+  text-align: right;
+  margin-top: 2rem;
+}
+
+.formField__control {
+ flex: 1;
+ height: 5rem;
+ outline: none;
+ margin-top: 2rem;
+ font-size: large;
+ border:0.1rem solid #eeeeee;
+ border-radius: 0.5rem;
+ padding: 0rem 1.5rem;
+}
+
+.formField__textField{
+  height: 15rem;  
+  flex: 1;
+  resize:vertical;
+  outline: none;
+  margin-top: 2rem;
+  font-size: inherit;
+  border:0.1rem solid #eeeeee
+}
+.cancelButton{
+  padding: 1.2rem 2rem;
+  background-color: #fafafa;
+  font-size: inherit;
+  color: black;
+  text-align: center;
+  float: right;
+  border: solid 0.05rem #e2e2e2;
+  border-radius: 0.4rem;
+  margin:5rem 0rem;
+}
+
+.cancelButton:hover{
+  cursor: pointer;
+}
+
+.submitButton{
+  padding: 1.2rem 2.9rem;
+  background-color: #dc8905;
+  font-size: inherit;
+  color: black;
+  text-align: center;
+  float: right;
+  color: white;
+  border: solid 0.05rem #e2e2e2;
+  border-radius: 0.4rem;
+  margin:5rem 5.5rem 1rem 3rem;
+}
+.submitButton:hover{
+  cursor: pointer;
+}
+.titleForm{
+  font-size: 1.8rem
+}
+.activityCategory{
```

Comment:
For cleaner code, I think we should always have one line space before defining our class/id(s) in CSS